CIP-XXXX/README.md

```diff
@@ -0,0 +1,100 @@
+---
+CIP: XXXX
+Title: Cardano dApp-Wallet Network Magic Extension
```

Comment:
```suggestion
Title: Web-Wallet Bridge - Network Magic Extension
```
The word `Cardano` will always be redundant in a CIP title.  The term `Extension` might also be redundant & we can talk about that when it's introduced at today's meeting.

Reply:
@nathanbogale p.s. the CIP meeting attendees today couldn't think of any additions to the CIP-0030 API that aren't "extensions" of some kind: effectively ruling that `Extension` is redundant with the classification `Web-Wallet Bridge`.

I added that if `Network Magic` itself doesn't characterise the extension properly that some other word might be added or substituted (e.g. `Network Magic Determination` or even just `Network Determination`).